Fix link to cheatsheet in help page

glostis opened the pull request:
The link to the cheatsheet is broken when browsed using `:help` (see attached screenshot). This PR fixes this - at least locally.

![2021-01-29-121929_747x240_scrot](https://user-images.githubusercontent.com/25295717/106269123-63f4c200-622c-11eb-941c-98251d1e26c9.png)

The-Compiler reviewed (changes requested):
Thanks! While this fixes the issue, it's a somewhat confusing fix: The `link:` isn't actually needed for the original `https://` link, because asciidoc already treats `https:...` as link.

However, the reason this is breaking is actually the link rewriting in `scripts/asciidoc2html.py`.  There, in `_build_docs`, there's the replacement with a `qute://` link, which will need this `link:...` prefix to work properly.

Could you please fix this there instead?

glostis commented:
I wasn't aware of this link substitution, I have implemented the fix you mentioned.

I also removed a `link:` that was prepending the same URL in `quickstart.asciidoc`.

The problem is that my fix now breaks the link associated to the `cheatsheet-small.png` image in `quickstart.asciidoc`, as the link gets replaced by `link:qute://help/img/cheatsheet-big.png`, but stays that way in the HTML, `asciidoc` does not remove the `link:` from it.
Do you think this is a bug in `asciidoc`, or something I didn't deal with correctly?

The-Compiler commented:
Thanks for the update! Let me note it might take me a bit until I get back to you because I'm currently busy with a pytest training. I'll probably get this into v2.0.2 since it is a bugfix though.

glostis commented:
Sure, no problem! The inconvenience of the bug is very minor 😄

The-Compiler commented:
Actually I took another quick look :smile: 

Unfortunately my suggestion earlier wasn't correct - like you've seen as well, replacing the link like I suggested would also break the link inside the image... I've taken the freedom to apply your original fix instead as b4b22b765e8143754a2f6023e437b7d10c915cc0 (master) and 18fed6109bec6ef8b34a165087375f746191c9ee (v2.0.x).

Thanks for the contribution and apologies for the confusion!
